packages/gatsby-plugin-vanilla-extract/README.md

```diff
@@ -0,0 +1,51 @@
+# gatsby-plugin-vanilla-extract
+
+Gatsby plugin which wraps the [vanilla-extract](https://vanilla-extract.style/) project for easy use in Gatsby.
+
+## Setup
+
+`npm install gatsby-plugin-vanilla-extract @vanilla-extract/babel-plugin @vanilla-extract/css @vanilla-extract/webpack-plugin`
```

Comment:
````suggestion
```shell
npm install gatsby-plugin-vanilla-extract @vanilla-extract/babel-plugin @vanilla-extract/css @vanilla-extract/webpack-plugin
```
````

Always better to put install messages in big code blocks